src/websocket.js

```diff
@@ -139,8 +139,8 @@ class WebSocket extends EventTarget {
   }
 
   send(data) {
-    if (this.readyState === WebSocket.CLOSING || this.readyState === WebSocket.CLOSED) {
-      throw new Error('WebSocket is already in CLOSING or CLOSED state');
+    if (this.readyState === WebSocket.CONNECTING) {
+      throw new DOMException("Failed to execute 'send' on 'WebSocket': Still in CONNECTING state");
```

Comment:
use the base Error as DOMException does not exist in node

Reply:
please change it to `Error` (the 1st variant) to support current node versions.
But you can also add a comment like `// TODO: node>=17 replace with DOMException`